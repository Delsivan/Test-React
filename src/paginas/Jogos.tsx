import React from "react"
import Formulario from "../componentes/Formulario"
import ListaPessoas from "../componentes/ListaPessoas"
import Rodape from "../componentes/Rodape"

const Jogos = () => {
    return (
        <>
            <Formulario />
            <ListaPessoas />
            <Rodape />
        </>
    )
}

export default Jogos