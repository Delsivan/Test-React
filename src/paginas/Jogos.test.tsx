import { screen, render, fireEvent } from "@testing-library/react"
import React from "react"
import { RecoilRoot } from "recoil"
import Jogos from "./Jogos"

const mockNavegacao = jest.fn()

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavegacao
    }
})

describe('pagina jogos', () => {
    test('deve renderizar corretamente', () => {
        const { container } = render(
            <RecoilRoot>
                <Jogos />
            </RecoilRoot>
        )

        expect(container).toMatchSnapshot()
    })
})