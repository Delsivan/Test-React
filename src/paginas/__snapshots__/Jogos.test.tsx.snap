// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pagina jogos deve renderizar corretamente 1`] = `
<div>
  <form>
    <input
      placeholder="Digite os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Iniciar
    </button>
  </footer>
</div>
`;
